compiler/modules/comm.diagnostic/diagnostic.bal

```diff
@@ -118,11 +113,9 @@ public function messageFormat(Template t) returns string {
 public function format(Diagnostic d) returns string[] {
     Location loc = d.location;
     string line = loc.file.filename();
-    LineColumn? lc = locationLineColumn(loc);
-    if lc != () {
-        // lineColumn returns 0-based lines currently
-        line += ":" + lc[0].toString() + ":" + (lc[1] + 1).toString();
-    }
+    LineColumn lc = locationLineColumn(loc);
+    // lineColumn returns 0-based lines currently
+    line += ":" + lc[0].toString() + ":" + (lc[1] + 1).toString();
     line += ": error: " + d.message;
```

Comment:
No need for a `line` variable: just use `return` with a concatenation expression.

Reply:
fixed